manifests/init.pp

```diff
@@ -281,6 +284,7 @@
   Boolean $manage_service                         = true,
   Boolean $service_enable                         = true,
   Enum['running', 'stopped'] $service_ensure      = 'running',
+  Hash $service_override                          = {},
```

Comment:
If the EPP is `Hash[String, String]`, I'd make it the same here. I'd also consider `String[1]` to only allow non-empty strings, at least for the keys.